src/lib/crypto/krb/prng_os.c

```diff
@@ -91,3 +91,11 @@ krb5_c_random_make_octets(krb5_context context, krb5_data *outdata)
     }
     return 0;
 }
+
+krb5_error_code KRB5_CALLCONV
+krb5_c_random_os_entropy(krb5_context context, int strong, int *success)
+{
+    /* The OS RNGs that accept input from userspace do not increase the
+     * entropy estimate based on this new entropy. */
```

Comment:
This comment confuses me.  krb5_c_random_os_entropy() doesn't contribute any entropy from the application (except the timing of the call, I guess), so there's no reason to expect an increase in the estimated entropy of the OS PRNG.